_data/wallets.yml

```diff
@@ -38,6 +38,8 @@ websites:
       btc: Yes
       othercrypto: Yes
       doc: https://breadwallet.com/blog/breadwallet-plan-bitcoin-cash-bch/
```

Comment:
is this still valid? Didnt Breadwallet change to Bread?

Reply:
I can change the name to "Bread", but the URL is still valid if that's what you're asking, and I believe it will continue to be.

Reply:
no they offer a wallet I thought? maybe we need to split them?